core/src/components/modal/modal.scss

```diff
@@ -57,6 +57,20 @@ ion-backdrop {
   pointer-events: auto;
 }
 
+/**
+ * When focus trap is disabled on a visible modal, allow clicks to pass through
+ * to content behind it. The .modal-wrapper retains pointer-events: auto so
+ * modal content remains interactive. We only apply this when show-modal is
+ * present to avoid affecting hidden modals that haven't presented.
+ */
+:host(.ion-disable-focus-trap.show-modal) {
+  pointer-events: none;
+}
+
+:host(.ion-disable-focus-trap.show-modal) ion-backdrop {
+  pointer-events: none;
+}
```

Comment:
```suggestion
:host(.ion-disable-focus-trap.show-modal),
:host(.ion-disable-focus-trap.show-modal) ion-backdrop {
  pointer-events: none;
}
```

Reply:
Sorry, I'm still iterating on this. I thought I had it last night, but it didn't fix the reproductions. I think I've fixed the reproductions, but at this time I'm trying to apply a better solution than what I attempted here. I've converted to a draft for now to prevent confusion